templates/tftmpl/template_funcs.go

```diff
@@ -42,6 +45,22 @@ func joinStringsFunc(sep string, values ...string) string {
 	return strings.Join(cleaned, sep)
 }
 
+// hclServicesFunc is a wrapper of the template function to marshal Consul
+// catalog service tag information into HCL. It returns the list of tags with
+// formatted like: "["tag1", "tag2"]". It returns an empty array string "[]"
+// when no tags
+func hclServicesFunc() func(tags *dep.ServiceTags) string {
```

Comment:
`HCLServices` and `hclServicesFunc()` function names are strongly tied to the `services` template function name, but it was not intuitive for me to realize this formats just the tags for a single service until reading the signature. Could you rename this to reflect the input/output of the function and relate it to "tags"? The "services" portion of the name makes it seem like the parameter should be tags for multiple services, perhaps something like `hclServiceTagsFunc()`

Reply:
Good point, the name matches the template function name but isn't intuitive. I like hclServiceTagsFunc()!  Changed!